Jenkinsfile

```diff
@@ -26,7 +26,7 @@ pipeline {
     }
     environment {
         DOCS = 'distribution/docs'
-        ITESTS = 'distribution/test/itests/test-itests-ddf'
+        ITESTS = 'distribution/test/itests'
```

Comment:
✏️ Is this property needed anymore now that there is an itest profile?

Reply:
Yeah, we use it for the itests build. The "itests" profile determines whether the itests are included in the build, but if you just ran `mvn install -Pitests`, it would include all the other modules in the project as well.

To run _only_ the itests, you have to use the `-pl` flag and specify exactly which modules you want to build - `mvn install -Pitests -pl distribution/test/itests/test-itests-ddf`